### PR TITLE
Raincoat rework fix + Perforators chances

### PR DESCRIPTION
**Pull request by Damglador**

### CODE REVIEW

**Comment by Damglador** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by xoxfaby — CHANGES_REQUESTED** [review]
Could you include English strings as well?

**Comment by Damglador** [comment]
> Could you include English strings as well?

Ok

**Comment by Damglador** [comment]
Im done

File: ModComponents/ItemStats.cs [hunk 1]
```diff
@@ -190,7 +192,7 @@ public static void Initialize()
             RegisterStat(DLC1Content.Items.HalfSpeedDoubleHealth, "BETTERUI_MOVEMENTSPEED", 0.5f, HyperbolicStacking);
             RegisterModifier(ItemTag.MovementSpeed, DLC1Content.Items.HalfSpeedDoubleHealth, ItemModifier.ExponentialBonus, 0.5f);
             RegisterStat(DLC1Content.Items.HealingPotion, "BETTERUI_USES", 1f, statFormatter: StatFormatter.Charges);
-            RegisterStat(DLC1Content.Items.ImmuneToDebuff, "BETTERUI_MAXHEALTH", 100, statFormatter: StatFormatter.HP, itemTag: ItemTag.MaxHealth);
+            RegisterStat(DLC1Content.Items.ImmuneToDebuff, "BETTERUI_RAINCOAT", 1f, stackingFormula: LinearStacking, statFormatter: StatFormatter.Charges);
```
Comment (xoxfaby) [inline ModComponents/ItemStats.cs:195]: This should probably use the Raincoat item name, not a custom betterui string, see other items as example

**Review by xoxfaby — CHANGES_REQUESTED** [review]

File: ModComponents/ItemStats.cs (hunk 1 above)
Comment (xoxfaby) [reply ModComponents/ItemStats.cs:195]: Actually it should probably say "Debuffs Blocked" ?

Comment (xoxfaby) [reply ModComponents/ItemStats.cs:195]: and use a new StatFormatter for Debuffs? Charges also kinda makes sense

**Pull request merged**